corehq/ex-submodules/phonelog/utils.py

```diff
@@ -108,7 +108,9 @@ def _process_user_error_subreport(domain, xform):
             xform_id=xform.form_id,
             i=i,
             app_id=error['app_id'],
-            version_number=int(error['version']),
+            # beta versions have 'version', but the name should now be
+            # 'app_build'.  Probably fine to remove after June 2016.
```

Comment:
Yeah, but that version will never be live, it's only in 2.27, and I assume that anyone currently using the testing build of that will have moved on by then.  This will be changed before 2.27 gets officially released.